pkg/cluster/k8sres.go

```diff
@@ -1048,11 +1048,13 @@ func (c *Cluster) getNumberOfInstances(spec *acidv1.PostgresSpec) int32 {
 	cur := spec.NumberOfInstances
 	newcur := cur
 
-	/* Limit the max number of pods to one, if this is standby-cluster */
 	if spec.StandbyCluster != nil {
-		c.logger.Info("Standby cluster can have maximum of 1 pod")
-		min = 1
-		max = 1
+		if newcur > 1 {
+			c.logger.Warningf("operator only supports standby clusters with 1 pod")
+		} else {
+			min = newcur
+			max = newcur
```

Comment:
What is the idea behind these assignments ? This change effectively enables shutting down standby clusters (`newcur == 0` ) in production by circumventing the restriction imposed by the `min_instances` param ; that shutting down  might as well happen by mistake

Reply:
fair point. Now, I'm only setting it to the the spec's value if it's one.